Scripts/Modules/MultipleRayInputModule/MultipleRayInputModule.cs

```diff
@@ -386,20 +386,23 @@ private void Select(GameObject go)
 
 		private GameObject GetRayIntersection(RaycastSource source)
 		{
-			GameObject hit = null;
 			// Move camera to position and rotation for the ray origin
 			m_EventCamera.transform.position = source.rayOrigin.position;
 			m_EventCamera.transform.rotation = source.rayOrigin.rotation;
 
-			RayEventData eventData = source.eventData;
+			var camera = U.Camera.GetMainCamera();
```

Comment:
Add a comment OR is this even necessary? Do we need to change far clip or was it really just about adjusting the near clip to a reasonable value?